poetry/core/masonry/utils/package_include.py

```diff
@@ -66,7 +66,7 @@ def check_elements(self):  # type: () -> PackageInclude
         else:
             if root.is_dir():
                 # If it's a directory, we include everything inside it
-                self._package = root.name
+                self._package = self._include.replace("/", ".")
```

Comment:
@sondrewb one last question; is platform specific path seperators an issue here?

Reply:
Any updates here?

Reply:
@abn If it is an idea, I can work on this and finish it off next week if @sondrewb does not have time for it. let me know

Reply:
Happy to entertain a new pr. Maintaining the current change attributions of course. :)

Reply:
Of course :100:  Should I mention @sondrewb on the PR or you meant that I fork his project instead?

Reply:
Neither. Just rebase their commits onto your fork/branch. Simply retain the original commit author, if squashing use [`Co-Authored-By`](https://docs.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors).